api/controllers/file_processor/file-base64-upload.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * file_processor/file-base64-upload.js
+ *
+ *
+ * @api {post} /file/upload/base64/:objID/:fieldID Upload
+ * @apiParam {string: base64} file
+ * @apiParam {uuid} photoID
+ * @apiParam {uuid} objID
+ * @apiParam {uuid} fieldID
+ * @apiParam {string} uploadedBy
+ * @apiParam {number} size
+ * @apiParam {string} type
+ * @apiParam {string} fileName
+ * @apiUse successRes
+ * @apiSuccess (200) {Object} data
+ * @apiSuccess (200) {string} data.uuid
+ */
+
+const async = require("async");
+const { error } = require("console");
+
+var inputParams = {
+   file: { string: true, required: true },
+   photoID: { string: true, required: true },
+   objID: { string: true, required: true },
+   fieldID: { string: true, required: true },
+   uploadedBy: { string: true, required: true },
+   size: { number: true, required: true },
```

Comment:
Do we trust the client to provide an accurate size? It seems like we calculate it anyways on line 87?

Reply:
Was mostly copying params from old methods: a few of these may be trusting the client too much, but this one _is_ getting calculated anyway.